tests/acceptance/features/apiSharingNg1/sharedByMe.feature

```diff
@@ -3439,3 +3439,430 @@ Feature: resources shared by user
         }
       }
       """
+
+
+  Scenario: sharer (also a group member) lists the file and folder shares shared to group (Personal space)
+    Given group "grp1" has been created
+    And user "Alice" has been added to group "grp1"
+    And user "Alice" has created folder "FolderToShare"
+    And user "Alice" has uploaded file with content "hello world" to "textfile.txt"
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | textfile.txt |
+      | space           | Personal     |
+      | sharee          | grp1         |
+      | shareType       | group        |
+      | permissionsRole | Viewer       |
+    And user "Alice" has sent the following resource share invitation:
+      | resource        | FolderToShare |
+      | space           | Personal      |
+      | sharee          | grp1          |
+      | shareType       | group         |
+      | permissionsRole | Viewer        |
+    When user "Alice" lists the shares shared by her using the Graph API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should contain resource "textfile.txt" with the following data:
```

Comment:
is their any reason for checking json separately for file and folder?

Reply:
Not any specific reason but I found that JSON was checked separately for file and folder in following scenario  :
``` feature
Scenario: sharer lists the file and folder shares (Personal space)
```

Reply:
should be fine rather than having long json which doesn't mention which file/folder we are checking

Reply:
>is their any reason for checking json separately for file and folder?
>>Not any specific reason 

because it's easier to see which resource we are checking and the step happens to exist already.